Source/ScrollingNavbar+Sizes.swift

```diff
@@ -20,9 +20,13 @@ extension ScrollingNavigationController {
     }
 
     var tabBarOffset: CGFloat {
-        // Only account for the tab bar if a tab bar controller is present and the bar is not translucent
+        // Only account for the tab bar if a tab bar controller is present and the bar is not hidden
         if let tabBarController = tabBarController {
-            return tabBarController.tabBar.translucent ? 0 : tabBarController.tabBar.frame.height
+            if tabBarController.tabBar.hidden {
+                return 0
+            } else {
+                return tabBarController.tabBar.translucent ? 0 : tabBarController.tabBar.frame.height
```

Comment:
Hmm, yes, it's break code when tab bar is translucent, but in old version it's break view when tab bar is hidden and not translucent; appears black rect at bottom. i'm think how fix this now.